src/erlcron.app.src

```diff
@@ -1,6 +1,6 @@
 {application,erlcron,
  [{description,"Erlang Implementation of cron"},
-  {vsn,"semver"},
+  {vsn,"0.1"},
```

Comment:
After merging I'll be updating this to go semantic versioning style, and tagging it as X.Y.Z semantic versioning

Reply:
The reason I took out the "semver" was to get rid of the rebar's plugin dependency. I believe rebar3 handles this stuff automatically if the repository has a version tag in semver format.  Perhaps it's worth converting the project to rebar3, since rebar is already obsolete.